compiler/damlc/tests/src/query-lf.jq

```diff
@@ -4,30 +4,29 @@ def resolve_interned_string(pkg): pkg.interned_strings[.];
 
 def resolve_interned_dname(pkg): pkg.interned_dotted_names[.] | .segments_interned_str | map(resolve_interned_string(pkg));
 
-def get_value_name(pkg): .name_interned_dname | resolve_interned_dname(pkg);
+def get_value_name(pkg): .name_interned_dname // 0 | resolve_interned_dname(pkg);
 
-def get_template_name(pkg): .tycon_interned_dname | resolve_interned_dname(pkg);
+def get_template_name(pkg): .tycon_interned_dname // 0 | resolve_interned_dname(pkg);
 
 def get_module_name(pkg): .module.module_name_interned_dname // 0 | resolve_interned_dname(pkg);
 
-def get_dotted_name(pkg): .name_interned_dname | resolve_interned_dname(pkg);
+def get_dotted_name(pkg): .name_interned_dname // 0 | resolve_interned_dname(pkg);
 
-def get_field(pkg): .field_interned_str | resolve_interned_string(pkg);
+def get_field(pkg): .field_interned_str // 0 | resolve_interned_string(pkg);
 
-def get_name(pkg): .name_interned_str | resolve_interned_string(pkg);
+def get_name(pkg): .name_interned_str // 0 | resolve_interned_string(pkg);
 
-def get_text(pkg): .text_interned_str | resolve_interned_string(pkg);
+def get_text(pkg): .text_interned_str // 0 | resolve_interned_string(pkg);
 
 def norm_ty(pkg): if has("interned") then pkg.interned_types[.interned] else . end;
 
 # @SINCE-LF 1.7
 def norm_qualified_module(pkg; f):
     .type | norm_ty(pkg) | .struct.fields |
     map(.type | norm_ty(pkg)) |
-    { "package":
-        .[0] |
-        (try (.struct.fields[0] | get_field(pkg)) catch null)
```

Comment:
For future readers: the problem was that adding the fallback `// 0` to the getter functions above meant that the `try` part of this `try-catch` didn't fail, so the tests were failing. We opted for entirely removing the `package` field since they are not useful for damlc integration tests. The features that they back (namely module reexports and orphan instance imports) are still tested through the data-dependencies test suite.